### PR TITLE
Juhlee/doc instruction update

### PR DESCRIPTION
**Pull request by nicepat**

### CODE REVIEW

File: XML/RegionAndStorageAccounts.xml [hunk 1]
```diff
@@ -2,106 +2,106 @@
 <AllRegions>
 	<eastasia>
 		<StandardStorage>eosgeastasia</StandardStorage>
-		<PremiumStorage>konkacieastasiaxiov2</PremiumStorage>
+		<PremiumStorage>eosgeastasiaxiov2</PremiumStorage>
```
Comment (iamshital) [inline XML/RegionAndStorageAccounts.xml]: replace 'xiov2' with only 'x' for all strings.

**Comment by nicepat** [comment]
@iamshital Can you look into the line number 49, 52, and 79 in 'How-to-use.md' file? I should have your input before merging this PR.

File: Documents/How-to-use.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+    Here is the format inside of TestCases.xml file. TODO: Revise the definition, and required field or not.
+        <testName></testName>: Represent Test Case name
+        <testScript></testScript>
+        <PowershellScript></PowershellScript>: Actual PS script file name.
+        <files></files>
+        <setupType></setupType>: The name represents VM's size and its definition in TestsConfigurations.xml file, VMConfigurations folder.
+        <TestType></TestType>
+        <TestFeature></TestFeature>
+        <SupportedExecutionModes></SupportedExecutionModes>: AzureServiceManagement is retired. Only suppot AzureResourceManager
```
Comment (iamshital) [inline Documents/How-to-use.md]: Please remove this.

File: Documents/How-to-use.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        2. CommunityTests.xml: Tests from Open Source Community.
+        3. FunctionalTests.xml: Feature tests for SR-IOV, GPU, DPDK, etc.
+        4. Other.xml: If any does not fall into existing Category, add to here.
+        5. PerformanceTests.xml: Performance test cases
+        6. RegressionTests.xml: Add any tests for regression cycle. 
+        7. SmokeTests.xml: It will run before BVT test runs.
+        8. StressTests.xml: Under development. Network traffic and stroage IO testing under heavy CPU and Memory stress.
+
+    Here is the format inside of TestCases.xml file. TODO: Revise the definition, and required field or not.
+        <testName></testName>: Represent Test Case name
```
Comment (iamshital) [inline Documents/How-to-use.md]: Put a comment to make this "unique"

File: Documents/How-to-use.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+## Objective
+
+    This document instructs for partners to create customized menu of Jenkins pipeline and run either/both Microsoft-provided tests or/and customized test cases in Jenkins. This instruction has 2 major parts; creating menu from xml files and executing tests.
+
+## Develop tests in GitHub
+
+`Source: https://github.com/LIS/LISAv2`
+
+    1. XML folder: it has pre-defined global configuration and account information. It also has Regision information in xml files. This folder also has two sub folders; TestCases and VMConfigurations. VMConfigurations has the list of xml files for each test case. TestCases folder has the list of xml files for each test category. The master branch is owned by Microsoft, and actively manage the PR in LISAv2. New test case development and/or new menu development must be approved by Microsoft.
```
Comment (iamshital) [inline Documents/How-to-use.md]: Please fix the typo "Regision"

File: Documents/How-to-use.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+        1. Sign in to Jenkins page with the assigned user name & password
+        2. Browse to '<Partner name>-Refresh-Test-Selection-Menu', if you would like to apply new menu or test cases before test execution. 
+        3. Click 'Build with Parameters' in left panel menu
+            a. Enter git repo ULR and branch name for menu xml file. Recommend keeping the default Repo URL and branch name.
+            b. Click 'Build' button.
+            c. Click the rotation icon of running job, and verify 'SUCCESS' inside 'Console Output'
+        4. Browse to '<Partner name>-Launch-Tests' and submit the job for test execution.
+        5. Click 'Build with Parameters' in left panel menu
+            a. Select 'ImageSource' or navigate to 'CustomVHD'. If you have external source URL, you can enter in 'CustomCHDURL' text box.
```
Comment (iamshital) [inline Documents/How-to-use.md]: Fix the typo "CustomCHDURL"

File: Documents/How-to-use.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+            b. Leave 'kernel' unless you would like to change to customized kernel code or linux-next.
```
Comment (iamshital) [inline Documents/How-to-use.md]: kernel --> "Kernel"

File: README.md [hunk 7]
```diff
@@ -38,18 +38,16 @@ There are two ways to authenticate your machine with your subscription.
       To learn more about how to configure your PowerShell with your subscription, please visit [here](http://azure.microsoft.com/en-us/documentation/articles/powershell-install-configure/#Connect).
 
 ### Download Public Utilities
-Download Putty executables from http://www.putty.org and keep them in `.\TestScripts\Tools`. You should have the following utilities:
+Download Putty executables from http://www.putty.org and keep them in `.\Tools`. You should have the following utilities:
```
Comment (iamshital) [inline README.md]: No need to download these tools.

File: README.md [hunk 8]
```diff
@@ -38,18 +38,16 @@ There are two ways to authenticate your machine with your subscription.
...
 
         � plink.exe
         � pscp.exe
-        � putty.exe
-        � puttygen.exe
 
-Download dos2unix executables from http://sourceforge.net/projects/dos2unix/ and keep them in `.\TestScripts\Tools`. You should have the following utilities:
+Download dos2unix executables from http://sourceforge.net/projects/dos2unix/ and keep them in `.\Tools`. You should have the following utilities:
```
Comment (iamshital) [inline README.md]: Remove all download tool instructions.

**Pull request merged**